client/src/main/java/org/threadly/litesockets/client/ws/WSClient.java

```diff
@@ -63,7 +63,7 @@
   private final HTTPStreamClient hsc;
   
   private volatile WebSocketDataReader onData;
-  private volatile WebSocketOpCode wsoc = WebSocketOpCode.Binary;
+  private volatile WSOPCode wsoc = WSOPCode.Binary;
```

Comment:
These names are not as obvious to me, just saying

Reply:
I went back andforth on just calling it OPCode, and letting the package name distinguish it, but seemed odd in implementation.  Not sure I like WebSocketOPCode much better though, and seems like a long name for something that in the end is just one byte.